appveyor.yml

```diff
@@ -0,0 +1,29 @@
+# AppVeyor.com is a Continuous Integration service to build and run tests under
+# Windows
+
+environment:
+
+    global:
+        CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
```

Comment:
What does this do?  I don't see where %CMD_IN_ENV% is being used.

Reply:
Probably nothing, I forgot to remove it after copying it from some similar configuration.